core/common/src/test/java/alluxio/ConfigurationTestUtils.java

```diff
@@ -89,9 +89,9 @@ public static void resetConfiguration() {
     conf.put(PropertyKey.MASTER_JOURNAL_TAILER_SHUTDOWN_QUIET_WAIT_TIME_MS, "50ms");
     conf.put(PropertyKey.MASTER_JOURNAL_TAILER_SLEEP_TIME_MS, "10ms");
 
-    // If tests fail to connect they should fail early rather than using the default ridiculously
-    // high retries
-    conf.put(PropertyKey.USER_RPC_RETRY_MAX_NUM_RETRY, "3");
+    // To keep tests fast, we should do more retries with a lower max wait time.
+    conf.put(PropertyKey.USER_RPC_RETRY_MAX_NUM_RETRY, "60");
+    conf.put(PropertyKey.USER_RPC_RETRY_MAX_SLEEP_MS, "500ms");
```

Comment:
Unrelated to this PR, we have deprecated the use of `_MS` and other time unit suffixed properties

Reply:
👍